.gitignore

```diff
@@ -0,0 +1,5 @@
+build/
+vendor/
+composer.lock
+composer.phar
```

Comment:
Why is all of this being added? `http-middleware` contains only interfaces and will never be testable, etc.